package.json

```diff
@@ -35,6 +35,7 @@
     "@types/react": "18.0.25",
     "@types/react-dom": "18.0.8",
     "@types/recharts": "^1.8.24",
+    "@typescript-eslint/eslint-plugin": "^5.42.1",
```

Comment:
question: what is this for?

Reply:
I was getting an error about this missing package before, but I tried just now and it's working without this. Removing now

Reply:
Removed

Reply:
Can we revert package-lock changes too?